.vsts-ci.yml

```diff
@@ -217,9 +243,14 @@ jobs:
       - BuildPackages
       agentOs: Windows_NT
       name: PublishToMyGet
+      displayName: Publish To MyGet
       enableTelemetry: $(_enableTelemetry)
       enablePublishBuildArtifacts: true
       helixRepo: dotnet/iot
+      strategy:
+        matrix:
+          release_configuration:
+            _BuildConfig: Release
```

Comment:
nit: you could also just set this as

```yaml
variables:
- _BuildConfig: Release
```

Reply:
true in this case since we only build the one configuration but I leave as is if that's ok for consistency with the other jobs which do have more than one configuration.